themes/aura/locales/fr.json

```diff
@@ -73,7 +73,9 @@
       "discount": "Réduction",
       "coupon": "Coupon",
       "coupon_applied": "Coupon appliqué avec succès",
-      "coupon_removed": "Coupon supprimé avec succès"
+      "coupon_removed": "Coupon supprimé avec succès",
+      "shipping": "Expédition",
```

Comment:
I think this should be "Livraison" based on the same translation as Classic theme's